presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -655,8 +665,12 @@ SLASH: '/';
 PERCENT: '%';
 CONCAT: '||';
 
-STRING
-    : '\'' ( ~'\'' | '\'\'' )* '\''
+UNICODE_STRING
+    : ESCAPE_INTRODUCER STRING
+    ;
+
+BASIC_STRING
```

Comment:
It seems strange that `string` is a `BASIC_STRING` or `UNICODE_STRING`, but `BASIC_STRING` is a `STRING`. 

I'd leave `STRING` as is, make `string` be `STRING` or `UNICODE_STRING`, and get rid of this `BASIC_STRING` rule.